manifests/base/kustomization.yaml

```diff
@@ -2,10 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
```

Comment:
I think that namespace can be removed also.
Eventually, we should control namespace by overlay [kustomization](https://github.com/kubeflow/tf-operator/blob/master/manifests/overlays/kubeflow/kustomization.yaml#L3)

WDYT @deepak-muley @Jeffwan ?

Reply:
Agree @andreyvelich . we should specify it only once in the top level kustomize. will remove it from here.